internal/asm/assembler.go

```diff
@@ -53,11 +53,63 @@ type ConstantValue = int64
 
 // StaticConst represents an arbitrary constant bytes which are pooled and emitted by assembler into the binary.
 // These constants can be referenced by instructions.
-type StaticConst = []byte
+type StaticConst struct {
+	Raw []byte
```

Comment:
Do I understand that raw means we don't know the encoding because it could be anything? ex string iee754 etc?

Reply:
Yeah can be any